app/views/admin/schedules/_day_tab.html.haml

```diff
@@ -1,18 +1,24 @@
+- cell_duration = EventType::LENGTH_STEP
```

Comment:
I would say `EventType::LENGTH_STEP` is already readable enough. And if not it would be better to rename it than having several variables that are the same thing.